pkg/pods/pod.go

```diff
@@ -367,17 +368,26 @@ func (pod *Pod) Install(manifest *PodManifest) error {
 		return util.Errorf("Could not setup config: %s", err)
 	}
 
-	launchables, err := pod.GetLaunchables(manifest)
-	if err != nil {
-		return err
-	}
+	for _, launchableStanza := range manifest.LaunchableStanzas {
+		launchable, err := pod.getLaunchable(launchableStanza)
+		if err != nil {
+			return err
+		}
 
-	for _, launchable := range launchables {
-		err := launchable.Install()
+		err = launchable.Install()
 		if err != nil {
 			pod.logLaunchableError(launchable.Id, err, "Unable to install launchable")
 			return err
 		}
+
+		err = cgroups.Default.SetCPU(launchable.Id, launchableStanza.CgroupConfig.Cpus)
+		if err != nil {
+			pod.logLaunchableError(launchable.Id, err, "Unable to set cgroup cpu limits")
+		}
+		err = cgroups.Default.SetMemory(launchable.Id, launchableStanza.CgroupConfig.Memory)
```

Comment:
We always generate cgroups at install time, but the default value will leave those groups completely unrestricted. Because the code is kinda scattered right now (cgroup generation in pod, cgexec invocation in hoistexecutable), it's easier than making the cgroup instantiation conditional